src/MusicStore/Get-Crossgen.ps1

```diff
@@ -32,9 +32,9 @@ function Get-CoreCLRVersion()
 
     foreach ($name in $json["libraries"].Keys)
```

Comment:
Are you only using the CoreClr package for crossgen?  If yes, you don't need to do this on 2.0 anymore. Cross gen comes in the tools folder of the runtime.RID.Microsoft.NETCore.App package now. So you can just restore your project for that RID, and grab cross gen from there.  The native Jit assembly comes in there too.